hs_access_control/models.py

```diff
@@ -2262,6 +2278,9 @@ def get_combined_privilege(self, this_user):
             raise HSAUsageException("Grantee is not a user")
         access_user = this_user.uaccess
 
+        if this_user.is_superuser:
+            return PrivilegeCodes.OWNER
+
```

Comment:
The access control migration has already created the uaccess object for the 'admin'